index.bs

```diff
@@ -191,11 +204,11 @@ The {{PerformanceEntry/name}} attribute's getter will return one of the followin
 Note: There are some inconsistencies across these names, such as the "-unreachable" and the "-contexts" suffixes.
 These names are kept for backward compatibility reasons.
 
-The {{PerformanceEntry/entryType}} attribute's getter will return <code>"longtask"</code>.
+The {{PerformanceLongTaskTiming/entryType}} attribute's getter step is to return <code>"longtask"</code>.
 
-The {{PerformanceEntry/startTime}} attribute's getter will return a {{DOMHighResTimeStamp}} of when the task started.
+The {{PerformanceLongTaskTiming/startTime}} attribute's getter step is to return a {{DOMHighResTimeStamp}} of when the task started.
 
-The {{PerformanceEntry/duration}} attribute's getter will return a {{DOMHighResTimeStamp}} equal to the elapsed time between the start and end of task, with a 1 ms granularity.
+The {{PerformanceLongTaskTiming/duration}} attribute's getter step is to return a {{DOMHighResTimeStamp}} equal to the elapsed time between the start and end of task, with a 1 ms granularity.
```

Comment:
[Existing issue] Do we need to specify how to round to 1 ms precision? Is it truncating, round-to-nearest, or something else?

Reply:
Actually, I like the wording in the definition used later in the spec (for LoAF timing). Perhaps something like:
```
The {{PerformancLongTaskTiming/duration}} attribute's getter step is to return the number of
milliseconds in the  [=duration=] between [=this=]'s {{PerformanceLongTaskTiming/startTime}}
and a {{DOMHighResTimeStamp}} of when the task ended.
```

(but again, this is existing text, essentially, so not blocking)